Steps/UploadCodeCoverage.yml

```diff
@@ -145,14 +153,22 @@ steps:
         cmd = f'{UPLOAD_CMD} -f {cov_file} -Z'
         if COV_FLAG:
           cmd += f' -F {COV_FLAG}'
-        process = subprocess.Popen(
-          cmd,
-          stdout=subprocess.PIPE,
-          stderr=subprocess.PIPE,
-          shell=True)
-        output, error = process.communicate()
-        print(f"##[debug]{output.decode('utf-8')}")
+
+        process = 0
+        for loop in range(0,5):
+          process = subprocess.Popen(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            shell=True)
+          output, error = process.communicate()
+          print(f"##[debug]{output.decode('utf-8')}")
+          if process.returncode == 0:
+            return
```

Comment:
This needs to be a break

Reply:
L152 we are looping through and submitting multiple reports. This return will cause an exit after the first report is submitted.